.github/scripts/notify-release-on-prs.ts

```diff
@@ -0,0 +1,155 @@
+import { Octokit } from '@octokit/rest';
```

Comment:
Consider adding an explanation section to better explain what this script does.
Also, wouldn't it be simpler to include the person who contributed the PR in the changelog and simply tag them as part of the release processon GitHub, i.e. mention them in the releases section.

Reply:
Added description to #3623.